aip/cloud/new.md

```diff
@@ -0,0 +1,135 @@
+---
+aip:
+  id: 2511
+  scope: cloud
+  state: draft
+  created: 2019-11-22
+permlink: /cloud/2511
+redirect_from:
+  - /2511
+---
+
+<!--
+TODO: discuss policy lifecycle? binding lifecycle?
+TODO: discuss policy conditions.
+TODO: Should the effective policy message be the same as the policy message? If it's different, should properties of an effective policy reference the concrete policy which caused it?
+-->
+
+# Policy API Guidance
+
+A policy is a statement of intent. When set on a
+[resource](https://aip.dev/121) it controls some behavior for that resource and
+its descendants. Some examples of controllable behaviors are:  who can access a
+resource, what configurations are allowed for a resource, and what billing
+account to charge for a resource’s usage.
+
+## Definitions
+
+There are 2 types of policies, _dedicated_ and _shared_.
+
+A _dedicated_ policy shares an identifier with the resource it is bound to. IAM
+policies in GCP are an example of dedicated polices. Users "set" the policy on
+directly on the resource they intend it to apply to, and all children of the
+resource inherit the policy.
+
+A _shared_ policy is a first-class resource (with its own identifier) and can
+even have policies applied to it (like an authorization policy). Unlike other
+resources, a shared policy can be _bound_ to one or more resources, and applies
+to those resources and any descendants. An example of a shared policy is
+billing accounts, which are associated with resources that bill to them.
+
+## Guidance
+
+### Standard Methods
+
+For most policy types, the policy name **should** be incorporated into the
+method name. For example, `GetBillingPolicy` for a Billing policy. The
+descriptions below omit the specific policy type for brevity.
+
+#### Dedicated Policies
+
+If a resource can only have one of your policy type bound to it, and there does
+not need to be a separation of authorization privilege between the resource and
+the policy, then you **may** use a dedicated policy.
+
+Dedicated polices are the simpler of the two to implement, with a smaller API
+scope. The authorization policy applied to the resource applies to the policy
+as well. This can be problematic if users have permissions to modify
+permissions on the resource, yet aren't intended to be able to modify the
+policy.
+
+- **SetPolicy** - Sets a dedicated policy for a resource
+- **GetPolicy** - Retrieves the dedicated policy for a resource
+- **DeletePolicy** - Deletes the dedicated policy for a resource
+- **LookupEffectivePolicy** - Retrieves the effective policy for a resource,
+  which is the combination of policies on the resource and any ancestors
+
+#### Shared Policies
+
+If you want to bind a single policy to multiple resources, or want to apply a
```

Comment:
Nit: We have not used second person much in the past. (It is not _wrong_, so much as just not the writing style of other authors...by which I mostly mean, well, me.) We may need a expanded AIP style guide at some point, because I feel guilty "correcting" this.

I propose "APIs" or "services" as a substitute for "you", purely for consistently. I acknowledge potential value in revisiting this. I am not convinced that either second-person or third-person is better, but I do want them to match.